### ISSUE TITLE
How to display multiple images in a loop ...?

### ISSUE THREAD

**Issue by breathe** [nteract/hydrogen#708]
Hi -- after upgrading from hydrogen 1.10.1 -> 1.11.0, I'm getting a different behavior when displaying multiple images in a loop ...

Here's a minimal code example.  I'm creating images in a loop and want to display all of the images in the cell output ...

```
from IPython.display import display, Image
import PIL.Image
from io import BytesIO
import cv2

def show_image(img, fmt='png'):
    a = np.uint8(img)
    f = BytesIO()
    PIL.Image.fromarray(a).save(f, fmt)
    display(Image(data=f.getvalue()))

test_images = glob.glob('./test_images/*')

for image in test_images[:2]:
    img = cv2.imread(image)
    show_image(img)
```

This was working in 1.10.1 showing all the images in the cell output.  In 1.11.I, I see the images displayed sequentially as the kernel runs then when the cell finishes executing, only the last image is visible for inspection ...  Is this a regression?  Is there an alternative way to accomplish my objective?

Thanks

**Comment by lgeiger** [comment]
@breathe Thanks for reporting. This looks like a regression slipped through in our latest release.

I wasn't able to reproduce your example but could mock a similar behavior with:
```python
import time
for i in range(10):
    print(i)
    time.sleep(1)
```

**Comment by lgeiger** [comment]
Fixed in `v1.11.1`
